CHANGELOG.md

```diff
@@ -14,9 +14,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   an extra path with modules.
 - `tt play`: support connection to a target instance by `application` name
   or `application:instance` name.
+- `tt coredump pack`: add several options to customize dump processing:
+  * `-e (--executable)`: specify Tarantool executable path.
+  * `-p (--pid)`: specify PID of the dumped process.
+  * `-t (--time)`: specify time of dump (seconds since the Epoch).
 
 ### Changed
 
+- `tt coredump pack`: by default tarantool executable path is obtained from
```

Comment:
Should it use a tarantool binary from the tt environment or not?

Reply:
It definitely should, but currently it's not. Let me resolve this in the scope of another issue (#1069).